syntax = "proto3";

service Showtime {
    rpc GetShowtimes(Empty) returns (stream ShowtimeData) {}
    rpc GetShowmovies(Date) returns (ShowtimeData) {}
}

message Date {
    string date = 1;
}

message ShowtimeData {
    string date = 2;
    repeated string movies = 3;
}

message Empty {
}